axum-extra/src/lib.rs

```diff
@@ -10,11 +10,12 @@
 //! Name | Description | Default?
 //! ---|---|---
 //! `async-read-body` | Enables the `AsyncReadBody` body | No
-//! `cookie` | Enables the `CookieJar` extractor | No
 //! `cookie-private` | Enables the `PrivateCookieJar` extractor | No
 //! `cookie-signed` | Enables the `SignedCookieJar` extractor | No
+//! `cookie` | Enables the `CookieJar` extractor | No
```

Comment:
Why did you move this down?

Reply:
Hm probably just vim sorting being weird